client/src/app/modules/model-manager/components/hugging-face-import-ribbon-content/huggingface-model-details/huggingface-model-details.component.html

```diff
@@ -14,6 +14,7 @@
       *ngIf="!(loading$ | async); else loading"
       [innerHTML]="markdownHTML$ | async"
     ></div>
+    <div class="readme-not-found wb-text-8" *ngIf="error$ | async as error">No readme found.</div>
```

Comment:
Do we use `error` obtained from `error$` observable anywhere?

Reply:
fixed